src/components/scenes/SendConfirmationScene.js

```diff
@@ -12,6 +12,7 @@ import { UniqueIdentifierModalConnect as UniqueIdentifierModal } from '../../con
 import { getSpecialCurrencyInfo } from '../../constants/indexConstants.js'
 import { intl } from '../../locales/intl'
 import s from '../../locales/strings.js'
+import { makeSpend } from '../../modules/Core/Wallets/api.js'
```

Comment:
I would like to stop using these pointless wrappers and just call core methods directly. `wallet.makeSpend(...)` is more obvious than `makeSpend(wallet, ...)`, and it's less code to maintain.

Reply:
So so agreed!

Reply:
Amen...Can we do it as an actual task rather than piece by piece. Basically get rid of the Wallets/api.js file and see where flow hates us.